### PR TITLE
tutorial workflow - added alternative wind obs data retrieval

### PR DESCRIPTION
**Pull request by wxtim**
Fixes part of the dependency on MetOffice datapoint.

- Get the wind CSV from recent observations stored in a fairly reliable (ogimet) third party source, leaving existing setup as a fallback.
- Get the observations lat-long data from a file in the repo.
- Swapped Belmullet for Aldergrove, because it's more available.
- Provided a mechanism to get an alternative obs site if the first one doesn't have wind data available.

### CODE REVIEW

**Review by MetRonnie — COMMENTED** [review]
The tutorial content needs updating (still has Belmullet) - https://cylc.github.io/cylc-doc/nightly/html/tutorial/scheduling/datetime-cycling.html#gathering-observations

Not sure what is happening when I'm running the `cylc/doc/etc/tutorial/cylc-forecasting-workflow` workflow. I'm getting a messed up wind plot image - https://github.com/cylc/cylc-doc/issues/275 (I posted this as a separate issue not realising I was still checked out on this branch. On master it's not producing the plot of course)

File: cylc/doc/etc/tutorial/cylc-forecasting-workflow/bin/get-observations [hunk 1]
```diff
@@ -107,17 +114,113 @@ def extract_observations(data):
...
+    obs = None
+    while dist < 1 and fails:
+        try:
+            obs = synop_grab(site_id, cycle_point)
+            fails = False
+        except NoDataException:
+            badsites.append(f'{int(site_id):05d}')
+            site_id, dist = get_nearby_site(site_id, badsites)
+
+    if obs == None:
```
Comment (kinow) [inline cylc/doc/etc/tutorial/cylc-forecasting-workflow/bin/get-observations:214]:
```suggestion
    if obs is None:
```

**Review by kinow — APPROVED** [review]
Don't know how to run the code, but reading the code it looks OK. Found one thing that probably needs fixing, but everything else is :ok: :ok_man: 

Thanks!

**Comment by wxtim** [comment]
> The tutorial content needs updating (still has Belmullet) - https://cylc.github.io/cylc-doc/nightly/html/tutorial/scheduling/datetime-cycling.html#gathering-observations

Good spot. I've done some sedding, both to turn "Belmullet" ⇒ "Aldergrove" but to fix the WMO numbers too.

File: cylc/doc/etc/tutorial/cylc-forecasting-workflow/bin/get-observations [hunk 2]
```diff
@@ -107,17 +114,113 @@ def extract_observations(data):
...
+
+    # See https://allaboutweather.tripod.com/synopcode.htm for an explanation
+    # of what this regex is looking for: Synopcode dd and ff
+    regex_wind = re.compile(
+        r'AAXX [\d\/]{5} [\d\/]{5} [\d\/]{5} [\d\/]'
+        r'(?P<direction>\d\d)(?P<speed>\d\d) 1'
+    )
+    try:
+        data = regex_wind.search(req.text).groupdict()
+    except:
```
Comment (MetRonnie) [inline cylc/doc/etc/tutorial/cylc-forecasting-workflow/bin/get-observations:160]:
```suggestion
    except AttributeError:
```

**Review by MetRonnie — COMMENTED** [review]
Still getting messed up wind plots - see the screenshot in #275

**Comment by wxtim** [comment]
> Still getting messed up wind plots - see the screenshot in #275

Gotit! (I think). Two missing minus signs in the string.split() indices.

**Review by MetRonnie — APPROVED** [review]
Those lat/lon were indeed the wrong way round, but I've still got the wind plot bug. Happy to merge this and reopen #275

**Pull request merged**